apps/backend-e2e/src/e2e/auth.cy.ts

```diff
@@ -15,140 +15,87 @@ describe('AUTHENTIFICATION TESTS', () => {
     });
   });
-  it('Auth -> Check self-profile of admin user with correct jwt.', () => {
+  it('Connecting with incorrect user.', () => {
     cy.request({
-      method: 'GET',
-      url: `/api/users/${idUserAdmin}`,
-      failOnStatusCode: false,
-      auth: {
-        bearer: jwtUser,
+      method: 'POST',
+      url: '/api/auth/login',
+      body: {
+        email: faker.internet.email(),
+        password: faker.internet.password(),
       },
+      failOnStatusCode: false,
     }).then((response) => {
-      expect(response.status).to.eq(200);
-      expect(response.body.id).to.eq(idUserAdmin);
-      expect(response.body.role).to.eq('ADMIN');
-      expect(response.body.active).to.eq(true);
-      expect(response.body.email).to.eq('admin@example.org');
+      expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Connecting with proper editor user but wrong password.', () => {
+  it('Connecting with proper user admin but wrong admin password.', () => {
     cy.request({
       method: 'POST',
       url: '/api/auth/login',
       body: {
-        email: 'editor@example.org',
-        password: 'unknown_user_pw',
+        email: 'admin@example.org',
+        password: faker.internet.password(),
       },
       failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Connecting correctly with editor user.', () => {
+  it('Login as admin', () => {
+    cy.login(Cypress.env('admin_credentials'));
+  });
+  it('Connecting with proper editor user but wrong password.', () => {
     cy.request({
       method: 'POST',
       url: '/api/auth/login',
       body: {
         email: 'editor@example.org',
-        password: 'editor',
+        password: 'unknown_user_pw',
       },
       failOnStatusCode: false,
     }).then((response) => {
-      expect(response.status).to.eq(201);
-      jwtUser = response.body.access_token;
-      idUserEditor = response.body.user_id;
-    });
-  });
-  it('Auth -> Check self-profile of editor user with wrong jwt.', () => {
-    cy.request({
-      method: 'GET',
-      url: `/api/users/${idUserEditor}`,
-      failOnStatusCode: false,
-      auth: {
-        bearer: 'incorrect_jwt',
-      },
-    }).then((response) => {
-      expect(response.status).to.eq(403);
+      expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Check self-profile of editor user with correct jwt.', () => {
-    cy.request({
-      method: 'GET',
-      url: `/api/users/${idUserEditor}`,
-      failOnStatusCode: false,
-      auth: {
-        bearer: jwtUser,
-      },
-    }).then((response) => {
-      expect(response.status).to.eq(200);
-      expect(response.body.id).to.eq(idUserEditor);
-      expect(response.body.role).to.eq('EDITOR');
-      expect(response.body.active).to.eq(true);
-      expect(response.body.email).to.eq('editor');
-    });
+  it('Login as an editor', () => {
+    cy.login(Cypress.env('editor_credentials'));
```

Comment:
No expect here ?

Reply:
This is a Cypress command, if the command fails, so does the test, so an expect statement would be superfluous.